testgres/api.py

```diff
@@ -7,27 +7,27 @@
 edit configuration files, start/stop cluster, execute queries. The
 typical flow may look like:
 
->>> with get_new_node() as node:
+>>> with get_new_node('test') as node:
 ...     node.init().start()
 ...     result = node.safe_psql('postgres', 'select 1')
 ...     print(result.decode('utf-8').strip())
 ...     node.stop()
-<testgres.node.PostgresNode object at 0x...>
+PostgresNode('test', port=..., base_dir=...)
```

Comment:
I believe that `'test'` should be prefixed by `name=`.

Reply:
I personally disagree as it makes the whole output 5 symbols larger, but I can make my peace with that : )